### DISCUSSION TITLE
sponsorblock impementation

### DISCUSSION THREAD

**Discussion by Bogo25** [pystardust/ytfzf#707] [Ideas]
Hello, would it be possible to implement sponsorblock in the program?

**Comment by Euro20179** [comment]
Use the [mpv sponsorblock](https://github.com/po5/mpv_sponsorblock) extension for mpv

**Reply by Bogo25** [reply]
Thank you. I was searching for sponsorblock and ytfzf, because of that I wasn't able to find it.
